packages/web/src/components/play-bar/mobile/PlayBar.tsx

```diff
@@ -214,7 +214,7 @@ const PlayBar = ({
                   variant='premium'
                   text={messages.preview}
                   coloredWhenLocked
-                  iconSize='small'
+                  iconSize='2xs'
```

Comment:
This change feels unrelated. Just a relevant use of the new size?

Reply:
the context here is that it had it's own pattern where small meant 12px, so just updated it to use harmony values for consistency